### ISSUE TITLE
Create a strategy-ish pattern architecture for `/ptu` autocompletes

### ISSUE THREAD

**Issue by beanc16** [beanc16/roll-of-darkness-bot#11]
Currently, command execution for all `/ptu` subcommands is handled with a strategy-ish pattern (a handler map / an object with handler functions indexed by enum strings, abstracted behind a class that takes the enum key(s) as input). This same pattern should be created for autocompletion for the `/ptu` command in a way that keeps the "strategies" separate from the command execution strategies (or includes them with the relevant strategies as a separate method).

Once this is done, evaluate whether or not a separate issue should be made for other autocomplete commands, or if they can be done small enough that they can be batched together with this issue.

**Comment by beanc16** [comment]
Autocomplete was moved to the strategy executor here: https://github.com/beanc16/roll-of-darkness-bot/commit/7b835bdce74604d088c543e87e6ea6a0dcffba37

Next, this just needs to be separated out into specific strategies

**Comment by beanc16** [comment]
After moving autocomplete to strategy executors, I decided that doing a strategy-ish pattern would be overabstraction since each if statement was just one function call. Instead, I opted to use a handler map pattern within the strategy executors.

This was completed in the following commits:
- I created enums for the parameter names of all autocomplete parameters: https://github.com/beanc16/roll-of-darkness-bot/commit/ef2f9175088827e6dbc3b569641ced710631bef8
- I implemented the handler map pattern, which should add type safety in order to ensure that we always have a way of handling any autocomplete values we add to our enums: https://github.com/beanc16/roll-of-darkness-bot/commit/b546abd8a3df8b2b7a3c7531c5fccfed73f0a373
- Then moved the autocomplete handling for `/cb` to the strategy executor in the same way that `/ptu` and `/nwod` handle it here: https://github.com/beanc16/roll-of-darkness-bot/commit/8748ddaed5328c520c7a1048a2b9c631a407aec7
